MapboxNavigation/CarPlayManager.swift

```diff
@@ -95,7 +95,16 @@ public protocol CarPlayManagerDelegate {
      - returns: A bool indicating whether to disable idle timer when carplay is connected and enable when disconnected.
      */
     @objc optional func carplayManagerShouldDisableIdleTimer(_ carPlayManager: CarPlayManager) -> Bool
-
+    
+    /**
+     Called when the carplay manager will present the favorites list.
+     
+     Implementing this method will allow developers to present a custom list of favorites triggered by the default trailing bar button.
+     
+     - parameter carPlayManager: The carplay manager that will present the list.
+     - returns: A `CPListTemplate` containing a list of favorites.
+     */
+    @objc optional func carPlayManagerFavoritesList(_ carPlayManager: CarPlayManager) -> CPListTemplate
```

Comment:
This doesn't explicitly need to be a favorites list in the delegate protocol. It should be bundled into the trailing template items instead.